libsource/src/main/java/com/virtusize/libsource/network/VirtusizeApiTask.kt

```diff
@@ -189,28 +254,30 @@ internal class VirtusizeApiTask(
     /**
      * Check if the response of the API request is a JSON array
      * @param apiRequestUrl The input stream of bytes
+     * @return the boolean value to tell whether the response of the apiRequestUrl is a JSON array.
      */
     private fun responseIsJsonArray(apiRequestUrl: String): Boolean {
         return apiRequestUrl.contains(VirtusizeEndpoint.ProductType.getPath())
                 || apiRequestUrl.contains(VirtusizeEndpoint.UserProducts.getPath())
     }
 
     /**
-     * Returns the contents of an InputStream as a String.
+     * Returns the contents of an [InputStream] as a String.
      * @param inputStream The input stream of bytes
+     * @return the string from scanning through the inputStream
      */
     private fun readInputStreamAsString(inputStream: InputStream): String? {
         val scanner: Scanner = Scanner(inputStream).useDelimiter("\\A")
         return if (scanner.hasNext()) scanner.next() else null
     }
 
     /**
-     * Returns the VirtusizeError that is associated with an API error
-     * @param urlPath The endpoint path of an API request
-     * @param responseCode The response code of an API request
-     * @param response The response from an API request
+     * Gets the API error message based on the path part of the request url
+     * @param requestPath the path part of the request URL
+     * @param response the response from an API request
+     * @return the message with the info of the request's path and response
      */
-    private fun virtusizeNetworkError(urlPath: String?, responseCode: Int?, response: Any?): VirtusizeError {
-        return VirtusizeError(VirtusizeErrorType.NetworkError, responseCode, "$urlPath - ${response?.toString()}")
+    private fun getAPIErrorMessage(requestPath: String?, response: Any?): String {
```

Comment:
@gpark-vs here I changed it to getAPIErrorMessage

Reply:
👍